examples/bpk-component-fieldset/examples.js

```diff
@@ -524,4 +538,5 @@ export {
   DisabledSelectExample,
   DisabledCheckboxExample,
   MixedExample,
+  SplitInputExample,
```

Comment:
We should probably add an example for split input component outside the fieldset too as it can be used on its own too?

Reply:
Yes it can be used on it's own as well. I added it as an example under fieldset since it was the recommended placement and following the example of the other components in fielset. Will add an example :)

Reply:
Yep, it makes sense to have it as an example under fieldset!
but as with select, input etc. it should have an example of its own too :)